lib/cursor.js

```diff
@@ -8,7 +8,7 @@ class Cursor extends Readable {
 
     // Currently mongodb cursor options itcount, readPref, showDiskLoc are not implemented - and some other
     // See https://docs.mongodb.com/manual/reference/method/js-cursor/
-    const operations = ['batchSize', 'hint', 'limit', 'maxTimeMS', 'max', 'min', 'skip', 'snapshot', 'sort']
+    const operations = ['batchSize', 'hint', 'limit', 'maxTimeMS', 'max', 'min', 'skip', 'snapshot', 'sort', 'collation'];
```

Comment:
Can we document this somewhere? Looks like https://github.com/mongoist/mongoist#cursor might list the other options here.

Reply:
Good place to note that it's only supported on 3.4 and up too.

Reply:
Done